### ISSUE TITLE
The 'truncation' strategy does not exist for the mongoid ORM! (1.5.0)

### ISSUE THREAD

**Issue by hackeron** [DatabaseCleaner/database_cleaner#390]
This seems to be similar to #299 but unfortunately the workaround of including mongoid-tree in the Gemfile no longer works :(

```
$ rspec
Run options: include {:focus=>true}

All examples were filtered out; ignoring {:focus=>true}

Randomized with seed 19705


Finished in 0.00751 seconds (files took 0.86593 seconds to load)
0 examples, 0 failures

Randomized with seed 19705

/usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/base.rb:139:in `rescue in orm_strategy': The 'truncation' strategy does not exist for the mongoid ORM!  Available strategies: truncation (DatabaseCleaner::UnknownStrategySpecified)
	from /usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/base.rb:135:in `orm_strategy'
	from /usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/base.rb:33:in `create_strategy'
	from /usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/base.rb:57:in `strategy='
	from /usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/base.rb:157:in `set_default_orm_strategy'
	from /usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/base.rb:11:in `initialize'
	from /usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/configuration.rb:29:in `new'
	from /usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/configuration.rb:29:in `add_cleaner'
	from /usr/local/lib/ruby/gems/2.2.0/gems/database_cleaner-1.5.0/lib/database_cleaner/configuration.rb:23:in `[]'
	from /Users/Hackeron/Development/Xanview/timeline/spec/spec_helper.rb:64:in `block (2 levels) in <top (required)>'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/example.rb:378:in `instance_exec'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/example.rb:378:in `instance_exec'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/hooks.rb:357:in `run'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/configuration.rb:1660:in `block in run_hooks_with'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/configuration.rb:1660:in `each'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/configuration.rb:1660:in `run_hooks_with'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/configuration.rb:1626:in `with_suite_hooks'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:114:in `block in run_specs'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/reporter.rb:77:in `report'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:113:in `run_specs'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:89:in `run'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:73:in `run'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:41:in `invoke'
	from /usr/local/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.2/exe/rspec:4:in `<top (required)>'
	from /usr/local/lib/ruby/gems/2.2.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:268:in `load'
	from /usr/local/lib/ruby/gems/2.2.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:268:in `block in load'
	from /usr/local/lib/ruby/gems/2.2.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /usr/local/lib/ruby/gems/2.2.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:268:in `load'
	from /usr/local/lib/ruby/gems/2.2.0/gems/spring-commands-rspec-1.0.4/lib/spring/commands/rspec.rb:18:in `call'
	from /usr/local/Cellar/ruby/2.2.2/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /usr/local/Cellar/ruby/2.2.2/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
```

**Comment by hackeron** [comment]
Ah, it seems the gem is broken and missing the mongo2 directory :( - as a workaround, this works:

```
gem 'database_cleaner', git: 'https://github.com/DatabaseCleaner/database_cleaner.git'
```

**Comment by benhutton** [comment]
yeah, looks like a gemspec bug. mongo2 files need to be added here: https://github.com/DatabaseCleaner/database_cleaner/blob/v1.5.0/database_cleaner.gemspec#L116-L120

(and maybe the gemspec shouldn't try to enumerate all the files??)

**Comment by etagwerker** [comment]
This is fixed over here: https://github.com/DatabaseCleaner/database_cleaner/pull/385

Once that gets merged I'll close this.

**Comment by etagwerker** [comment]
@hackeron Could you test with `master` and let me know?

**Comment by dukex** [comment]
@etagwerker using master works here! Thank you!

**Comment by francois2metz** [comment]
This works! Any chance to release a new version to fix this? Thanks.

**Comment by etagwerker** [comment]
Fixed in version 1.5.1.
